demo/src/static/code-markdown/icon.md

```diff
@@ -0,0 +1,131 @@
+```
```

Comment:
This needs to be a longer discussion - unlike luma.gl, these demos are not standalone applications, and a lot of the code wrapped into the components do not make sense unless you know how the container uses them. I don't like where it is either - it's hard to maintain and very confusing for people who try to dig into the demo source. I'm willing to make a push to refactor this before v4.